src/middlewares/whitelist.ts

```diff
@@ -0,0 +1,20 @@
+import { Request, Response, NextFunction } from 'express';
+import { getRequestInfo } from '../utils/users';
+import { isAuthenticated } from './authentication';
+import whitelist from '../../whitelist.json';
+import { forbidden } from '../utils/responses';
+import { Error } from '../types';
+
+export default [
+  ...isAuthenticated,
+  (request: Request, response: Response, next: NextFunction) => {
+    const { user } = getRequestInfo(response);
+    const { tournamentId } = request.body;
+    if (tournamentId in whitelist) {
+      return whitelist[<keyof typeof whitelist>tournamentId].includes(user.email)
+        ? next()
+        : forbidden(response, Error.NotWhitelisted);
```

Comment:
```suggestion
      if(whitelist[<keyof typeof whitelist>tournamentId].includes(user.email))
        return next()
      return forbidden(response, Error.NotWhitelisted);
```

Cette syntaxe est un peu plus claire, et on n'ajoute pas de lignes de code